cms/documents/migrations/0001_initial.py

```diff
@@ -28,7 +28,7 @@ class Migration(migrations.Migration):
                 ("title", models.CharField(max_length=255)),
                 ("file", models.FileField()),
                 ("created_at", models.DateTimeField(auto_now_add=True)),
-                ("file_size", models.PositiveIntegerField(editable=False, null=True)),
```

Comment:
@nehakerung and undo this migration too :D

Reply:
🫡 [dab2db0](https://github.com/ONSdigital/dis-wagtail/pull/43/commits/dab2db074966db2b9e38932202f054a1f4af3382)